desktop/src/containers/ForemanProject/projectSummary.container.tsx

```diff
@@ -86,19 +86,19 @@ export const ForemanProjectSummary = (props: Props) => {
       <div className="flex flex-row flex-grow justify-between bg-slate-50 border border-slate-100 p-4 rounded-md">
         <Typography>Hours Worked Last Week:</Typography>
         <Typography>
-          {projectSummary.hoursWorkedLastWeek}
+          {Math.round(projectSummary.minutesWorkedLastWeek / 60)}
         </Typography>
       </div>
       <div className="flex flex-row flex-grow justify-between bg-slate-50 border border-slate-100 p-4 rounded-md">
-        <Typography>Hourse Worked This Week:</Typography>
+        <Typography>Hours Worked This Week:</Typography>
         <Typography>
-          {projectSummary.hoursWorkedThisWeek}
+          {Math.round(projectSummary.minutesWorkedThisWeek / 60)}
```

Comment:
I think this will give us "Hours Worked This Week: 12" when I also want to include the minutes like: 
"Hours Worked This Week: 12:45"

Reply:
<img width="659" alt="CleanShot 2023-08-19 at 12 42 16@2x" src="https://github.com/joshuawootonn/time-track/assets/20950876/f52d3567-40b0-4d2d-a5d8-476cd54d164a">
For this example we want

Reply:
6:15 | 423:45 | 187:45